Doc/library/argparse.rst

```diff
@@ -323,7 +323,7 @@ argument to :class:`ArgumentParser`::
 
    >>> parser = argparse.ArgumentParser(
    ...     description='A foo that bars',
-   ...     epilog="And that's how you'd foo a bar")
+   ...     epilog='And that's how you'd foo a bar')
```

Comment:
This is a syntax error.

Reply:
Oh you are right. I must have missed that.

```suggestion
   ...     epilog="And that's how you'd foo a bar")
```